apps/crashing.go

```diff
@@ -45,6 +53,69 @@ var _ = AppsDescribe("Crashing", func() {
 		})
 	})
 
+	Describe("an app with three instances, two running and one crashing", func() {
+		It("keeps two instances running while another crashes", func() {
+			By("Pushing the app with three instances")
+			Expect(cf.Cf(
+				"push", appName,
```

Comment:
Some parameters are already defined in the test app's manifest, do we need both?

Reply:
Not really..